### PR TITLE
Make error handling more consistent

### PR DESCRIPTION
**Pull request by geomacy**
Initial changes for making error handling more consistent (send all error output to stderr).

Includes updates to 
- access
- activity
- application
- catalog

commands.

### CODE REVIEW

File: api/activities/activities.go [hunk 1]
```diff
@@ -7,41 +7,32 @@ import (
...
 
-func ActivityStream(network *net.Network, activity, streamId string) string {
+func ActivityStream(network *net.Network, activity, streamId string) (string,  error) {
 	url := fmt.Sprintf("/v1/activities/%s/stream/%s", activity, streamId)
 	body, err := network.SendGetRequest(url)
-	if err != nil {
-		fmt.Println(err)
-	}
-	return string(body)
+	return string(body), err
```
Comment (lloyddave) [inline api/activities/activities.go:37]: Need to be careful here, as it is likely that string var `body` will not have been created.  Safer to just return empty string `""`.

File: api/application/applications.go [hunk 2]
```diff
@@ -8,139 +8,114 @@ import (
 )
 
 //WIP
-func Fetch(network *net.Network) string {
+func Fetch(network *net.Network) (string, error) {
 	url := "/v1/applications/fetch"
 	body, err := network.SendGetRequest(url)
 	if err != nil {
-		fmt.Println(err)
+		return string(body), err
```
Comment (lloyddave) [inline api/application/applications.go:15]: Need to be careful here, as it is likely that string var body will not have been created. Safer to just return empty string "".

Comment (geomacy) [inline api/application/applications.go:15]: Good catch.  As there are numerous places with this in the code can we just accept this and other such pull requests, and I will go through the codebase on a second pass when they are all merged into master, and fix these cases.

File: api/application/applications.go [hunk 3]
```diff
@@ -8,139 +8,114 @@ import (
...
+func Descendants(network *net.Network, app string) (string, error) {
 	url := fmt.Sprintf("/v1/applications/%s/descendants", app) 
 	
 	body, err := network.SendGetRequest(url)
-	if err != nil {
-		fmt.Println(err)
-	}
 	// TODO return model
-	return string(body)
+	return string(body), err
```
Comment (lloyddave) [inline api/application/applications.go]: Need to be careful here, as it is likely that string var body will not have been created. Safer to just return empty string `""`.

File: api/application/applications.go [hunk 4]
```diff
@@ -8,139 +8,114 @@ import (
...
+func DescendantsSensor(network *net.Network, app, sensor string) (string, error) {
 	url := fmt.Sprintf("/v1/applications/%s/descendants/sensor/%s", app, sensor) 
 	
 	body, err := network.SendGetRequest(url)
-	if err != nil {
-		fmt.Println(err)
-	}
 	// TODO return model
-	return string(body)
+	return string(body), err
```
Comment (lloyddave) [inline api/application/applications.go]: Need to be careful here, as it is likely that string var body will not have been created. Safer to just return empty string `""`.

File: api/application/applications.go (hunk 2 above)
Comment (lloyddave) [inline api/application/applications.go:15]: Agreed.  Merging.

**Pull request merged**